index.html

```diff
@@ -174,16 +175,20 @@
 
         if (currentTime.getSeconds() === 0) {
             switch(currentTime.getMinutes()) {
-                case 0: chime0q.play(); chime0q.addEventListener('ended', chimeHourly); break;
```

Comment:
Here was the problem. The chimes beginning at xx:00:00 (and not the first hour strike).